test/puppetlabs/puppetdb/testutils.clj

```diff
@@ -33,7 +33,11 @@
     `(do ~@body)
     (let [case-versions (remove keyword? (take-nth 2 bindings))]
       `(doseq ~bindings
-         (testing (str "Testing case " '~case-versions)
+         (testing (str "Testing case: " (clojure.string/join
+                                          ", "
+                                          (map #(str %1 ": " %2)
+                                               '~case-versions
+                                               (list ~@case-versions))))
```

Comment:
@mruzicka hey, just wanted to ping you to see what you thought about the latest incarnation.